docs/release-notes/index.md

```diff
@@ -12,3 +12,26 @@ permalink: /release-notes
 ### v1.0.0 (released 2019-04-09)
 
 * First GA, production-ready release.
+
+## v2.0
+
+### v2.0.0 (released tbd)
+
+* Request specs use backwards-incompatible sets-arg-as syntax. Script to upgrade [here](https://github.com/square/spincycle/util/reformat-yaml/).
+
+  Old syntax:
+  ```
+  node-name:
+    category: {job | sequence | conditional}
+    type: node-type
+    sets: [new-arg]
```

Comment:
Maybe add a line with `...` between type and sets? It kind of looks like category, type, and sets are the only fields you can set for a node.